src/RadixDlt.NetworkGateway.GatewayApi/gateway-api-schema.yaml

```diff
@@ -4745,12 +4832,24 @@ components:
                 - kind: "U32"
                   value: "1"
     StateKeyValueStoreDataRequestKeyItem:
+#TODO PP: check if that's not a breaking change.
+      oneOf:
```

Comment:
I think this change is separate from the rest of the PR?

I think I'm in agreement with Krzysztof - this will break client generation:
* Different clients chose to deserialize `oneOf` differently
* In general we have avoided `oneOf` without discriminators because I think because this is inefficient with deserialization as they need to analyze / validate the payload to work out the type)

I would suggest it would be more useful to add comments to explain one or the other can be provided :). This is definitely one of those endpoints where more comments would be useful. But also happy to do this in a separate PR.

Reply:
ok i'll extract that into separate PR, assumed that it will be straightforward so I did it here.

Reply:
https://github.com/radixdlt/babylon-gateway/pull/780